react/modules/gaEvents.ts

```diff
@@ -8,8 +8,7 @@ import {
   getImpressions,
   getDiscount,
 } from './utils'
-
-export const shouldMergeUAEvents = () => Boolean(window?.__gtm__?.mergeUAEvents)
+import shouldMergeUAEvents from './utils/shouldMergeUAEvents'
```

Comment:
I extracted it to [its own file](https://github.com/vtex-apps/google-tag-manager/pull/102/files#diff-a8ea90752b146e88d1f36f775db12187831cedefb6753d6ecc61c471a8e002bb) because it's easier to mock it to [return true](https://github.com/vtex-apps/google-tag-manager/pull/102/files#diff-391092895397e693ba67e1580e4058bbe69f9c6340f18687e5c385731d7a7946R127) in the tests this way.

I had tried using [this partial module mock approach](https://jestjs.io/docs/mock-functions#mocking-partials) but it doesn't work in this case because this function [is used internally in this file](https://github.com/vtex-apps/google-tag-manager/pull/102/files#diff-1ea8976bf29d0502073e0dd4b20555afe500d68ee35b13c5001f048facbff50dR107). See [this SO answer](https://stackoverflow.com/questions/51269431/jest-mock-inner-function/55193363#55193363) for more details.